tests/components/ResultsCount.test.tsx

```diff
@@ -114,7 +122,22 @@ describe('Results count for universal search', () => {
 
   it('Renders nothing if there is no result', () => {
     mockAnswersState(mockedStateUniversalNoResult);
+    const { container } = render(<ResultsCount />);
+    expect(container.childNodes[0]).toBeEmptyDOMElement();
```

Comment:
Is there no other way to check that there's no text or if it's an empty div beside drilling into the container? I would rather not going into the specifics of a component's dom element if possible. If not, can we do container.firstChild instead?

Reply:
I tried firstChild, but I think it was upset since it wasn't a Testing Library method.  It said it didn't like direct node access. I can look into any other ways!

Reply:
I tried an alternative method using regex, let me know if its OK!

Reply:
Unfortunately after looking around, I've had no success finding a better way.  It generally does not like firstChild and gives a red squiggly, but it is OK with childNodes[0].  Is that OK, or should I continue the hunt?